test/Test/BaseDAO.hs

```diff
@@ -0,0 +1,297 @@
+-- SPDX-FileCopyrightText: 2020 TQ Tezos
+-- SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+module Test.BaseDAO
+  ( test_BaseDAO
+  ) where
+
+import Universum
+
+import qualified Data.Map.Strict as Map
+import Lorentz
+import Lorentz.Test
+import Morley.Nettest
+import Test.Tasty (TestTree, testGroup)
+import Test.Tasty.HUnit (testCase)
+import Util.Named
+
+import qualified Lorentz.Contracts.Spec.FA2Interface as FA2
+import qualified Lorentz.Contracts.BaseDAO as DAO
+-- import qualified Indigo.Contracts.SampleFA2 as DAO -- TODO: Remove after BaseDAO passed all the tests
+
+{-# ANN module ("HLint: ignore Reduce duplication" :: Text) #-}
+
+originateBaseDAO
+  :: MonadNettest caps base m
+  => m ((Address, Address), (Address, Address), TAddress FA2.Parameter, Address)
+originateBaseDAO = do
+  owner1 :: Address <- newAddress "owner1"
+  operator1 :: Address <- newAddress "operator1"
+  owner2 :: Address <- newAddress "owner2"
+  operator2 :: Address <- newAddress "operator2"
+
+  admin :: Address <- newAddress "admin"
+
+  let bal = Map.fromList
+        [ ((owner1, 0), 100)
+        , ((owner2, 0), 100)
+        ]
+  let (operators :: DAO.Operators) = DAO.mkOperators
+          owner1 operator1
+          owner2 operator2
+  let
+    originateData = OriginateData
+      { odFrom = AddressResolved genesisAddress
+      , odName = "BaseDAO"
+      , odBalance = toMutez 0
+      , odStorage = DAO.mkStorage admin bal operators
+      , odContract = DAO.baseDAOContract
+      }
+  dao <- originate originateData
+  pure ((owner1, operator1), (owner2, operator2), dao, admin)
+
+test_BaseDAO :: TestTree
+test_BaseDAO = testGroup "tests to check BaseDAO contract functionality"
+  [ testGroup "Operator:"
+      [ testCase "allows zero transfer from non-existent operator" $
+          nettestTestExpectation zeroTransferScenario
```

Comment:
I think we can temporarily define 
```
ignore :: Expectation -> Expectation
ignore _ = pass
```

and use it for all failing tests, we will remove it in #6 as soon as everything is properly is implemented. This way we can merge this PR (after making other requested changes) and not wait for #6.

Reply:
I've added this, as well as other suggestion you've made, Ivan. Let me know if this MR is good enough to be merged.